.travis.yml

```diff
@@ -81,7 +81,7 @@ jobs:
       dist: bionic
     - os: linux
       arch: amd64
-      env: TR_ARCH=centos
+      env: TR_ARCH=centos7
```

Comment:
do we still need to test centos7? Can we just change the version in the Dockerfile?

Reply:
> do we still need to test centos7?

Well... It is our only test OS which still uses Python 2 as default. Just today I saw an article about the RHEL 7.9 release (which is the last release in the RHEL 7 timeframe) which mentioned that it is supported until 2024. Although CentOS 7 is old it will stay for some time and I think there will still be users on that platform for a couple of years. It is also not broken. I think if we find something that does just not work on CentOS 7 we can think about not testing it on CentOS 7 any more, but right now, except for Travis test time, it does not really have a negative effect on us.

> Can we just change the version in the Dockerfile?

No. Because of Python 3, not all necessary packages are available in EPEL, we need to enable the Powertools repository for certain packages (and probably some other things I forgot).